.github/workflows/test.yml

```
@@ -0,0 +1,68 @@
+name: Create test builds
+
+on:
+  # Uncomment this if build on every push is needed
+  # push:
+  #   branches-ignore: [main, beta, alpha] # skip for deployment branches.
+  #   tags-ignore: ['*']
+  pull_request:
+
+env:
+  XCODE_VERSION: "14.0"
+  GITHUB_CONTEXT: ${{ toJSON(github) }} # used by fastlane for deploying app 
+  FASTLANE_XCODEBUILD_SETTINGS_RETRIES: 6 # increase timeout for xcodebuild settings. We have seen this command take a bit longer to do on CI server. 
+
+jobs:
+  deploy-android-app:
+    runs-on: macos-12
```

```suggestion
    runs-on: ubuntu-latest
```

Builds for android can run on linux. Linux builds are faster then mac.

---

I tried it but then bumped into errors so reverted it back. Can't recall the errors now but will try again to see what the errors were.